src/api/git.go

```diff
@@ -49,10 +50,15 @@ func GetWorkflowDescriptor(c *model.Config) {
 	wt, err := repo.Worktree()
 	utils.CheckIfError(err)
 
-	source := fmt.Sprintf("%s/%s", wt.Filesystem.Root(), c.Descriptor)
-	target := fmt.Sprintf("%s/%s", c.BaseDir, "workflow-descriptor.json")
+	defDescriptor, err := os.ReadFile("templates/default-descriptor.json")
+	utils.CheckIfError(err)
+	actDescrriptor, err := wt.Filesystem.Open(c.Descriptor)
```

Comment:
wt.Filesystem.Open(c.Descriptor) liefert ein File zurück, da fehlt noch das Schließen am Ende. Oder überseh ich da was? => zBsp.: defer actDescrriptor.Close() nach der Fehlerbehandlung.
Und in actDescrriptor sind zwei r :)